met/docs/Users_Guide/tc-pairs.rst

```diff
@@ -35,11 +35,11 @@ tc_pairs has required arguments and can accept several optional arguments.
 Required arguments for tc_pairs
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-1. The **-adeck source** argument indicates the adeck ATCF format data source containing tropical cyclone model forecast (output from tracker) data to be verified. It specifies the name of an ATCF format file or top-level directory containing ATCF format files ending in “.dat” to be processed. The **-adeck** or **-edeck** option must be used at least once.
+1. The **-adeck source** argument indicates the adeck TC-Pairs acceptable format data source containing tropical cyclone model forecast (output from tracker) data to be verified. Acceptable data formats are limited to the standard ATCF format and the one column modified ATCF file, generated from a genesis mode run for extratropical cyclones. It specifies the name of a TC-Pairs acceptable format file or top-level directory containing TC-Pairs acceptable format files ending in “.dat” to be processed. The **-adeck** or **-edeck** option must be used at least once.
```

Comment:
```suggestion
1. The **-adeck source** argument indicates the adeck TC-Pairs acceptable format data source containing tropical cyclone model forecast (output from tracker) data to be verified. Acceptable data formats are limited to the standard ATCF format and the one column modified ATCF file, generated by running the tracker in genesis mode. It specifies the name of a TC-Pairs acceptable format file or top-level directory containing TC-Pairs acceptable format files ending in “.dat” to be processed. The **-adeck** or **-edeck** option must be used at least once.
```

Reply:
I'm pretty sure that tracker can be run in genesis mode for tropical or extra-tropical cyclones. I don't think it's only for extra-tropical.

Reply:
True; I want to convey that the one-column modified file was only created to better track extratropical cyclones, though. Would users have any need to run Guang Ping's extra ID string creator for tropical cyclones? If they would, then your edit is the better wording.